go.mod

```
@@ -12,13 +12,15 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
+	github.com/elastic/beats/v7 v7.17.10
 	github.com/elastic/e2e-testing v1.99.2-0.20221205111528-ade3c840d0c0
 	github.com/elastic/elastic-agent-autodiscover v0.6.1
 	github.com/elastic/elastic-agent-client/v7 v7.1.2
-	github.com/elastic/elastic-agent-libs v0.3.8-0.20230512004837-d6a37d929cd6
+	github.com/elastic/elastic-agent-libs v0.3.9-0.20230606211614-a752b18efc9e
```

Now that there's a `0.3.9` release, I think you can just use that here.
```suggestion
	github.com/elastic/elastic-agent-libs v0.3.9
```